envs/environment.yaml

```diff
@@ -11,7 +11,7 @@ dependencies:
 - pip
 
 - atlite>=0.2.9
-- dask
+- dask<=2023.9.1
```

Comment:
For me, this doesn't need downgrading. Worked also with 2023.9.3.

Reply:
> For me, this doesn't need downgrading. Worked also with 2023.9.3.

> For me, this doesn't need downgrading. Worked also with 2023.9.3.

Great news.  According to the recent reports for pypsa-earth-sec, constraining `atlite` is enough. Dropped `dask` constraint.